File: FlowStudio/src/main/java/com/boraydata/workflow/utils/DateUtils.java
package com.boraydata.workflow.utils;


import java.text.Format;
import java.text.SimpleDateFormat;
import java.util.Date;

public class DateUtils {

    public static String toDate(Long time) {
        Date date = new Date(time);
        Format format = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
        return format.format(date);
    }
}

File: FlowAuth/src/main/java/com/boraydata/flowauth/utils/DateUtils.kt
package com.boraydata.flowauth.utils

import java.text.Format
import java.text.SimpleDateFormat
import java.util.*


object DateUtils {

    fun toDate(time: Long): String {
        val date = Date(time)
        val format: Format = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ")
        return format.format(date)
    }
}